swagger.yaml

```diff
@@ -689,12 +696,12 @@ definitions:
     description: "The group model for the response body"
     type: object
     properties:
+      id:
+        description: "The group id"
+        type: string
       name:
         description: "The group name"
         type: string
-      description:
-        description: "A description of the group"
-        type: string
       precedence:
         description: "The priority of the group"
         type: integer
```

Comment:
`GroupsList` on line 716 has the field name for the array of groups as `users` when it should be `groups`:

Current:
```
  GroupsList:
    description: "A list of groups for the response body"
    type: object
    properties:
      users:
        type: array
        items:
          $ref: '#/definitions/Group'
      count:
        type: integer
```
Expected:
```
  GroupsList:
    description: "A list of groups for the response body"
    type: object
    properties:
      groups:
        type: array
        items:
          $ref: '#/definitions/Group'
      count:
        type: integer
```